bindings/pom.xml

```diff
@@ -10,13 +10,13 @@
 	<parent>
 		<groupId>org.finos.symphony.toolkit</groupId>
 		<artifactId>symphony-java-toolkit</artifactId>
-		<version>6.20.0-SNAPSHOT</version>
+		<version>6.20.1-SNAPSHOT</version>
```

Comment:
Can we use the proper version instead of snapshot ?

Reply:
This is changed using versions plugin (see README.md).  it's best to keep it on snapshot for when we build locally.